doc/examples/viz_timers.py

```diff
@@ -0,0 +1,69 @@
+"""
+===============
+Using a timer
```

Comment:
Go this warning when running the example.

```
/home/macote/.local/venvs/dipy/lib/python3.6/site-packages/vtk/util/numpy_support.py:137: FutureWarning: Conversion of the second argument of issubdtype from `complex` to `np.complexfloating` is deprecated. In future, it will be treated as `np.complex128 == np.dtype(complex).type`.
  assert not numpy.issubdtype(z.dtype, complex), \
```

Reply:
Interesting. As far as I know we do not use complex numbers

Reply:
@skoudoro would be great if we can track down these FutureWarnings appearing when running this example.